CHANGELOG.md

```diff
@@ -7,29 +7,26 @@
 ### Bug fixes
 
 * `SyncUser.logout()` now correctly clears `SyncUser.currentUser()` (#3638).
-
-### Enhancement
-
-* `Realm.compactRealm()` works for encrypted Realms.
-*  Added `first(E defaultValue)` and `last(E defaultValue)` methods to `RealmList` and `RealmResult`. These methods will return the provided object instead of throwing an `IndexOutOfBoundsException` if the list is empty.
-*  Added `User.all()` that returns all known Realm Object Server users.
-
-### Credits
-
-* Thanks to Max Furman (@maxfurman) for adding support for `first()` and `last()` default values.
-
-## 2.0.3
-
-### Bug fixes
-
 * Those were not kept by ProGuard: names of native methods not in the `io.realm.internal` package, names of classes used in method signature (#3596).
 * Missing ProGuard configuration for libraries used by Sync extension (#3596).
 * Error handler was not called when sync session failed (#3597).
 * Permission error when a database file is located at external storage (#3140).
 
-### Enhancements
+### Enhancement
 
+* `Realm.compactRealm()` now works for encrypted Realms.
+* Added `first(E defaultValue)` and `last(E defaultValue)` methods to `RealmList` and `RealmResult`. These methods will return the provided object instead of throwing an `IndexOutOfBoundsException` if the list is empty.
 * Reduce transformer logger verbosity (#3608).
+* Added `User.all()` that returns all known Realm Object Server users.
+
+#Internal
```

Comment:
`### Internal`? Although some `Internal`s of `CHANGELOG.md` are H1, H2 or H3 randomly, but I think all `Internal`s should be H3 for consistency